bundle/config/mutator/process_target_mode_test.go

```diff
@@ -53,6 +54,9 @@ func mockBundle(mode config.Mode) *bundle.Bundle {
 				Models: map[string]*resources.MlflowModel{
 					"model1": {Model: &ml.Model{Name: "model1"}},
 				},
+				ModelServing: map[string]*resources.ModelServing{
```

Comment:
Given that the endpoints are named "serving endpoints", why use "model serving" here?

The API docs also mention "serving endpoints" ([doc](https://docs.databricks.com/api/workspace/servingendpoints)).

Reply:
Yeah great question, for Terraform, we received the guidance from ML Inference to call it `model_serving` [here](https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/model_serving) so I was just following that for consistency

Reply:
Synced offline, decided to rename to `model_serving_endpoints`.